R/module_data_summary.R

```diff
@@ -95,31 +94,46 @@ srv_data_summary <- function(id, teal_data) {
         } else if (is.null(summary_table_out)) {
           "no datasets to show"
         } else {
+          any_unsupported <- any(apply(summary_table_out, 1, function(x) all(is.na(x[-1]))))
+          if (any_unsupported) {
+            shinyjs::show("toggle_unsupported")
+          } else {
+            shinyjs::hide("toggle_unsupported")
+          }
```

Comment:
How about moving this to the separate observe? 
The rationale here is to keep it responsible for a single thing - rendering the table.
On the other hand: this is not very clear that rendering the summary table is doing show/hide the link

Reply:
Will do, thanks!